src/searchForm/stepper/overlay/index.tsx

```diff
@@ -1,21 +1,11 @@
 import styled from 'styled-components'
-import { color, radius, space } from '_utils/branding'
+import { overlayBaseStyle } from 'searchForm/baseStyles'
 
 import StepperOverlay from './StepperOverlay'
 
-const overlayWidth = '375px'
-
 const StyledStepperOverlay = styled(StepperOverlay)`
   & {
-    box-sizing: border-box;
-    width: ${overlayWidth};
-    border: 1px solid ${color.border};
-    border-radius: ${radius.l};
-    padding: ${space.xl};
-    padding-top: ${space.s};
-    /* Same as TripCard, we might need to have box-shadow specs so that we can move it to branding file */
-    box-shadow: 0 1pt 4pt rgba(0, 0, 0, 0.16), 0 2pt 8pt rgba(0, 0, 0, 0.08);
-    background: ${color.defaultBackground};
+    ${overlayBaseStyle}
```

Comment:
question: if you are already overriding the component SepperOverlay why do you need to use overlayBaseStyleHere?

Reply:
I don't override components I just factorized common elements and I call them in components ;)

Reply:
I meant ```const StyledComponent = styled(OtherComponent)``` as an override.
The question is that you could already apply this style to that component directly.. I guess not.